src/main/java/com/hca/car_dealership/CarDealershipApplication.java

```diff
@@ -0,0 +1,38 @@
+package com.hca.car_dealership;
+
+import com.hca.car_dealership.models.Vehicle;
+import com.hca.car_dealership.repositories.VehicleRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.util.List;
+
+@SpringBootApplication
+public class CarDealershipApplication implements CommandLineRunner {
+
+	@Autowired
+	private VehicleRepository vehicleRepository;
+
+
+	public static void main(String[] args) {
+		SpringApplication.run(CarDealershipApplication.class, args);
+	}
+
+	@Override
+	public void run(String... args) throws Exception {
+//		Scanner scanner = new Scanner(System.in);
+//		System.out.print("Enter lowest price of vehicle you are looking for: ");
```

Comment:
When you're ready the next step is to create a user interface to ask the user what they want to see